packages/core/test/foundry/optimistic-assertor/OptimisticAsserter.Ownership.t.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../fixtures/optimistic-assertor/SetUpOptimisticAssertor.sol";
+
+contract ContractTest is Test {
+    SetUpOptimisticAssertor.OptimisticAsserterContracts umaContracts;
+    OptimisticAssertor optimisticAssertor;
```

Comment:
note we could probs ditch this second state variable to be even cleaner but I just wanted to show how to access the other contracts from the fixture.